src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -40,6 +40,61 @@ abstract class Widget_Abstract extends \Tribe\Widget\Widget_Abstract {
 	 */
 	protected $view_slug;
 
+	/**
+	 * @todo update in TEC-3612 & TEC-3613
+	 *
+	 * {@inheritDoc}
+	 * {@inheritDoc}default
+	 */
+	public function setup() {
+		// Add the admin template class for the widget admin form.
+		$this->admin_template = tribe( Admin_Template::class );
+
+		// Setup the View for the frontend.
+		$this->setup_view();
+	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$instance" missing

<sub>hash: 97d58c30345f06848e89c65ccb3f3205</sub>